default.yml

```diff
@@ -797,6 +801,7 @@ RSpec/VariableDefinition:
   Enabled: true
 RSpec/VariableName:
   Enabled: true
+  - spec/requests/api_v2/metrics_spec.rb
```

Comment:
Let’s keep using `Exclude:` for patterns.
Instead, simply add such line at the top of `metrics_spec.rb`:
```
# rubocop:disable RSpec/VariableName